modules/ob-creating-a-build-with-oci-artifacts.adoc

```diff
@@ -0,0 +1,96 @@
+// This module is included in the following assembly:
+//
+// * work-with-builds/using-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="Creating-a-build-with-OCI-artifacts_{context}"]
+= Creating a build with OCI artifacts
+
+You can create a build using an Open Container Initiative(OCI) artifact as your source code. It is also known as scratch image. It is an empty base image used to create small and secure container images by adding only the application and its essential dependencies. You can pull the OCI artifact from a container registry and then push the resulting image to a target registry.
+
+
+.Prerequisites
+
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have installed the `oc` CLI.
+* Optional: You have installed the link:https://console.redhat.com/openshift/downloads[`shp` CLI].
+
+.Procedure
+
+. Create a `Build` resource and apply it to the {ocp-product-title} cluster by using the following example configuration:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: my-oci-build <1>
+spec:
+  source:
+    type: OCI
+    ociArtifact:
+      image: <quay.io/org/image:tag> <2>
+  strategy:
+    name: <strategy-name> <3>
+    kind: ClusterBuildStrategy
+  paramValues: 
+    - name: <parameter-name> <4> 
+      value: <parameter-value> <5>
```

Comment:
This is optional too; I will suggest removing them to minimize the confusion. Please see https://shipwright.io/docs/build/build/#configuring-a-build